source/common/protobuf/utility.cc

```diff
@@ -64,6 +64,7 @@ std::string MessageUtil::getJsonStringFromMessage(const Protobuf::Message& messa
   // Setting this option makes debugging easier because it keeps field names consistent in JSON
   // printouts.
   json_options.preserve_proto_field_names = true;
+  json_options.add_whitespace = true;
```

Comment:
This is used in various wire serializations. I think we probably don't want extra whitespace there universally. How about making this a bool arg that defaults false, and you can opt in stuff like admin pages?